musonant/codelab
# setup apollo-graphql

#### What does this PR do?
* Setup graphql and apollo client

#### Description of Task to be completed?
* Fetch data with graphql and apollo

#### How should this be manually tested?
* Clone the repo and start the app using `yarn start`
* On home page, a loader should be disapplied, after which content fetched from the api would be displayed on the screen

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories? (if applicable)
NA

#### Screenshots
![Simulator Screen Shot - iPhone Xs - 2019-06-26 at 10 58 19](https://user-images.githubusercontent.com/42358974/60170920-5af67000-9801-11e9-8b07-92f6ea6c7243.png)
